Libraries/Geolocation/RCTLocationObserver.m

```diff
@@ -41,7 +41,7 @@ + (RCTLocationOptions)RCTLocationOptions:(id)json
   return (RCTLocationOptions){
     .timeout = [RCTConvert NSTimeInterval:options[@"timeout"]] ?: INFINITY,
     .maximumAge = [RCTConvert NSTimeInterval:options[@"maximumAge"]] ?: INFINITY,
-    .accuracy = [RCTConvert BOOL:options[@"enableHighAccuracy"]] ? kCLLocationAccuracyBest : RCT_DEFAULT_LOCATION_ACCURACY
+    .accuracy = [RCTConvert BOOL:options[@"enableHighAccuracy"]] ? kCLLocationAccuracyBestForNavigation : RCT_DEFAULT_LOCATION_ACCURACY
```

Comment:
Did you notice any significant changes because of this? It seems to be really aimed at navigation (and "intended to be used only while the device is plugged in"), so I'm not sure it's a good idea to switch to this by default.